plugins/view-resources/src/components/Table.svelte

```diff
@@ -154,10 +154,31 @@
     query,
     (result) => {
       total = result.total
+      if (totalQuery === undefined) {
+        gtotal = total
+      }
     },
     { limit: 1, ...options, sort: getSort(_sortKey), lookup, total: true }
   )
 
+  const totalQueryQ = createQuery()
+  $: if (totalQuery !== undefined) {
+    totalQueryQ.query(
+      _class,
+      totalQuery ?? query,
```

Comment:
Looks like `totalQuery` is always not `undefined` here

Reply:
yes, no problem in this case :-) Will fix in some next PRs.